### ISSUE TITLE
JackHMMER search on gene pages - 404: Page not found

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#2291]
https://www.ebi.ac.uk/Tools/hmmer/

"To meet the ongoing demand for HMMER searches, as well as allow us to update the underlying sequence databases to more recent versions, we have accelerated the release of a new underlying search system. However, we have yet to completely address the issues of making large sequence alignments based on search results, as required by jackhmmer. As the generation of these alignments currently cause the web servers to crash, we have temporarily suspended jackhmmer searches. Also, based on current usage patterns we have omitted some target sequence databases. We anticipate returning most, if not all functionality, in the near future, but in the meantime please bear with us as we continue to upgrade the infrastructure behind the HMMER servers."

**Comment by ValWood** [comment]
Yep I'm aware of that, It's annoying because I used it all the time. We should probably remove this link until the new version is really. I keep offering to test but no response....

**Comment by kimrutherford** [comment]
> We should probably remove this link until the new version is really.

I've done that.

**Comment by kimrutherford** [comment]
> https://www.ebi.ac.uk/Tools/hmmer/

Latest news on that page:

-------------------------------

Announcement: New Hmmer Website

We will be transitioning to a new website architecture on May 20, 2025. We will not be running a parallel system, so you may have to make a few changes, especially if you are using the API.

The current website will be unavailable starting May 20, 2025. All search results obtained prior to the switch will not be accessible. This change will pave the way for a second upgrade to bring back Jackhmmer functionality.
We have introduced some breaking changes to the API. Please refer to the documentation.
For assistance contact us via the contact form. Thank you for your patience and understanding. We apologise for any inconvenience caused.

**Comment by kimrutherford** [comment]
From the news section of the HMMER page:

> Thanks! We're actively working on bringing back the jackhmmer functionality. We'll definitely post an update here announcing it which should happen in the following months.

https://bsky.app/profile/hmmer.bsky.social/post/3lpm3jyeqkc2l

**Comment by ValWood** [comment]
https://www.ebi.ac.uk/Tools/hmmer/search/jackhmmer
is back, so we can re-instate links

**Comment by kimrutherford** [comment]
> Reply from the EBI helpdesk:

It's been a few months so I've just sent a follow-up email.

**Comment by kimrutherford** [comment]
I've contacted the hmmer helpdesk again to ask if they have made any progress.

**Comment by kimrutherford** [comment]
> I've contacted the hmmer helpdesk again to ask if they have made any progress.

There was no reply in January or in May.

I've just tried them again.

We might have to give up on this.
